Lombiq.VueJs/Assets/Scripts/helpers/process-helpers.js

```diff
@@ -0,0 +1,38 @@
+const { handleErrorMessage } = require('nodejs-extensions/scripts/handle-error');
+const path = require('path');
+
+const args = process.argv.splice(2);
+const argumentOptions = args.length >= 2 ? JSON.parse(args[1]) : undefined;
+
+async function executeFunctionByCommandLineArgument(functions) {
+    if (!args[0]) return;
```

Comment:
1. Args is the command line arguments from `process.argv`, basically runtime constants.
2. It contains the name of the function, see [here](https://github.com/Lombiq/Orchard-Vue.js/blob/035011bd8ac3f5ff1eef535c4f49c59e6ec0d6fc/Lombiq.VueJs/package.json#L36). Optionally it could contain a JSON that will expand the `defaultOptions` (e.g. custom root path, further rollup aliases). Configuring the options is not needed now, but iirc we needed it in Finitive (with GE) and may need it in the future.
3. `{ "sfcRootPath": "Assets/Vue", "rollupAlias": { "batman": "../../../batman" } }` ([see traditional batman example](https://www.npmjs.com/package/@rollup/plugin-alias))

Reply:
I've refactored the function with more destructuring and explicit error messages to be cleaner and more developer-friendly.